src/domain/core/src/services/config.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use internal_error::{InternalError, ResultIntoInternal};
+use url::Url;
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+pub struct Config {
```

Comment:
The name is too generic. We have a lot of configs. This is maybe `ServerUrlConfig`

Reply:
Done